src/_derive/mod.rs

```diff
@@ -181,8 +181,10 @@
 //! - e.g. `#[clap(max_values(3))]` would translate to `arg.max_values(3)`
 //!
 //! **Magic attributes**:
-//! - `name = <expr>`: [`Arg::id`][crate::Arg::id]
+//! - `id = <expr>`: [`Arg::id`][crate::Arg::id]
 //!   - When not present: case-converted field name is used
+//! - `name = <expr>`: [`Arg::id`][crate::Arg::id]
+//!   - **Deprecated:** use `id = <expr>`
```

Comment:
Is it better to just leave this out?

I could backport this to clap 3 and add the deprecation there.

Reply:
Yes, all deprecated content is removed.  Remove this and add it back in in the backport